cmd/integrationArtifactUnDeploy.go

```diff
@@ -0,0 +1,103 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/SAP/jenkins-library/pkg/command"
+	"github.com/SAP/jenkins-library/pkg/cpi"
+	piperhttp "github.com/SAP/jenkins-library/pkg/http"
+	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/SAP/jenkins-library/pkg/telemetry"
+	"github.com/pkg/errors"
+)
+
+type integrationArtifactUnDeployUtils interface {
+	command.ExecRunner
+
+	// Add more methods here, or embed additional interfaces, or remove/replace as required.
+	// The integrationArtifactUnDeployUtils interface should be descriptive of your runtime dependencies,
+	// i.e. include everything you need to be able to mock in tests.
+	// Unit tests shall be executable in parallel (not depend on global state), and don't (re-)test dependencies.
+}
+
+type integrationArtifactUnDeployUtilsBundle struct {
+	*command.Command
+
+	// Embed more structs as necessary to implement methods or interfaces you add to integrationArtifactUnDeployUtils.
+	// Structs embedded in this way must each have a unique set of methods attached.
+	// If there is no struct which implements the method you need, attach the method to
+	// integrationArtifactUnDeployUtilsBundle and forward to the implementation of the dependency.
+}
+
+func newIntegrationArtifactUnDeployUtils() integrationArtifactUnDeployUtils {
```

Comment:
This and other methods are not used, correct? Could you remove those too please

Reply:
done...

Reply:
Done, but only for this method. The other two ...Utils methods are not used either, it seems to me.

Reply:
removed...